api/api/Roles/models.py

```diff
@@ -0,0 +1,23 @@
+from accounts.models import User
+from clubs.models import Club
+from django.db import models
+
+
+class Roles(models.Model):
+
+    CLUB_ROLE_TYPES = [
+        ('Coordinator', 'Coordinator'),
+        ('Co-Coordinator', 'Co-Coordinator'),
+        ('Core member', 'Core member'),
+        ('Faculty Incharge', 'Faculty Incharge'),
+        ('Convener', 'Convener'),
+        ('Co-convener', 'Co-convener'),
+        ('Conselor', 'Conselor'),
+        ('Associate-Conselor', 'Associate-Conselor')
+    ]
+
+    name = models.CharField(max_length=30, choices=CLUB_ROLE_TYPES)
+    club = models.ManyToManyField(Club, on_delete=models.CASCADE)
+    user = models.ManyToManyField(User, on_delete=models.CASCADE)
```

Comment:
These should be `ForeignKeyFields`. Why?

club as a `ManyToManyField` means, 1 object of club will have multiple objects of `Role` class, and 1 object of `Role` will have multiple clubs associated with it.

Do we want that? No.

One object of `Role` class means: one relationship between club, user and the role_name. Hence `ForeignKeyField` makes more sense.